packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenu.tsx

```diff
@@ -35,7 +35,7 @@ interface IContextMenuProps extends IMenuProps {
   folders: Array<DashboardFolder>;
   repos?: Array<DashboardRepo>;
   setRenaming: null | ((value: boolean) => void);
-  createNewFolder: () => void;
+  createNewFolder: (() => void) | null;
   createNewSandbox: () => void;
```

Comment:
Ready to go after this one!!

```suggestion
  createNewFolder: () => void;
  createNewSandbox: (() => void) | null;
```

Reply:
doh!! big difference